libs/wingsdk/src/std/resource.ts

```diff
@@ -506,6 +518,16 @@ export class Display {
    */
   public description?: string;
 
+  /**
+   * Whether the resource is user-named or compiler-named.
+   */
+  public type?: "user-named" | "compiler-named";
+
+  /**
+   * The name of the resource.
+   */
+  public name?: string;
```

Comment:
How is this different from Display.title?

Reply:
I can use the title instead, for sure 👍